cdap-ui/cypress/support/MultiRowWidgets/utilities.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export function propertySelector(property) {
+  return `[data-cy="${property}"]`;
```

Comment:
I like this function! :smile: 
Do you think renaming this to something that says this is `data-cy` selector would be helpful? When I first read the function name earlier in this PR, I thought this would select a plugin property.